### ISSUE TITLE
Option to disable animations

### ISSUE THREAD

**Issue by pmulholland42** [pmulholland42/spacecraft-map#35]
There should be a switch in the options pane that disables animations. The animations that need disabled are the options pane opening, the details pane opening, and the zooming/panning on the map.
